packages/desktop-client/src/components/reports/Header.tsx

```diff
@@ -129,7 +143,7 @@ export function Header({
           </SpaceBetween>
         </SpaceBetween>
 
-        <SpaceBetween>
+        <SpaceBetween gap={3}>
```

Comment:
Could we up this? I think it looks a bit squished.

Current:
<img width="475" alt="image" src="https://github.com/user-attachments/assets/215a7d9a-62e4-402e-8e40-e4ba20effb4e" />


Here's 10px
<img width="536" alt="image" src="https://github.com/user-attachments/assets/f28ba82b-e3da-42e2-8956-53cf352994a1" />

Reply:
small screens would have a problem.

Reply:
is there any way to force it to wrap when the save button starts to overlap with the filter?

Reply:
No idea. my knowledge of CSS is less than js

Reply:
No worries, we'll go with it for now then. If anyone works out a way it can be a separate PR